pulumi/src/statefulService.ts

```diff
@@ -379,9 +359,9 @@ export async function deployStatefulService(
         },
         spec: {
           entryPoints: ['web', 'websecure'],
-          routes: Object.entries(services)
-            .map(([service, { ports }]) =>
-              ports
+          routes: services
+            .map((service) =>
```

Comment:
think this got broken, `service` is now the object, not the name which is used when passing to `match`